doc/rtd/topics/faq.rst

```diff
@@ -200,6 +200,19 @@ systemd, please make sure to include the following logs.
         $ systemctl --failed
 
 
+Autoinstall, Preruncmd, Postruncmd
+==================================
+Since cloud-init ignores top level userdata cloud-config keys, other projects
+such as Juju and subiquity_ autoinstaller use a yaml-formatted config
+that combines cloud-init's userdata cloud-config yaml format with their custom
+yaml keys. These combined yaml configurations may be valid cloud-config files,
+however keys such as autoinstall, preruncmd, and postruncmd are not used by
+cloud-init to configure anything.
+
+Please direct bugs and questions related to the autoinstaller, Juju, and
```

Comment:
Shall we link to the support channels here? For Juju maybe https://discourse.charmhub.io/  and for autoinstaller either #subiquity channel on IRC or #ubuntu-server maybe. We should check on that with subiquity devs.
@dbungert any preferences on what mediums autoinstall questions get steered?

Reply:
+1 to linking the support channels

Reply:
There isn't a #subiquity channel today on IRC and based on the average feedback I have seen on IRC I don't plan to start one.  Others are certainly welcome.

Discourse and #ubuntu-server are the way to go today.

Reply:
+1 Thanks, I've added that to this PR.